distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.url.mvn.cfg

```diff
@@ -81,27 +96,63 @@ org.ops4j.pax.url.mvn.defaultRepositories=\
     file:${karaf.base}/${karaf.default.repository}@id=child.system.repository@snapshots
 
 #
+# if "defaultLocalRepoAsRemote" is set do *any* value, localRepository will be
```

Comment:
This is copied directly from https://github.com/apache/karaf/blob/karaf-4.1.1/assemblies/features/base/src/main/resources/resources/etc/org.ops4j.pax.url.mvn.cfg#L88, but I fix this typo anyway:+1: